Rollbar/Deploys/DeployApiCallResult.h

```diff
@@ -1,25 +1,19 @@
-//
-//  ApiCallResult.h
-//  Rollbar
-//
-//  Created by Andrey Kornich (Wide Spectrum Computing LLC) on 2018-09-17.
 //  Copyright © 2018 Rollbar. All rights reserved.
-//
 
 #import <Foundation/Foundation.h>
 
 #import "DeploymentDetails.h"
-#import "../RollbarJSONFriendlyObject.h"
+#import "RollbarJSONFriendlyObject.h"
 
-typedef enum DeployApiCallOutcome {
-    Outcome_Success,
-    Outcome_Error,
-} DeployApiCallOutcome;
+typedef NS_ENUM(NSInteger, DeployApiCallOutcome) {
```

Comment:
that is nice! looks like NS_ENUM allows much richer programmability. just learned something new...